packages/website/pages/docs/guides/prover-claim-ttko.mdx

```diff
@@ -0,0 +1,39 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Prover Claim TTKO
+
+## Overview
+
+This guide will help you claim TTKO earned as a prover.
```

Comment:
would recommend a prerequisites section. something like "need to have successfully proven a block, see here how to enable a prover".

Reply:
Update:
- You followed the [Enable a prover](/docs/guides/enable-a-prover) guide and have a prover which has proved a block to earn TTKO rewards.
- Must have a balance of ETH Sepolia (see: [Receive tokens](/docs/guides/receive-tokens)).